src/Config.ts

```diff
@@ -387,6 +403,23 @@ export interface ConfigParams {
    * - `ss`, `ss.s`, `ss.ss`, `ss.sss`, `ss.ssss`, etc. (seconds)
    *
    * The number of decimal places in the seconds token does not matter. All versions of the seconds token are equivalent in the context of parsing time values.
+   * Regardless of the time format specified, the hours-minutes-seconds value may be followed by the AM/PM designator.
+   *
+   * For more information, see the [Date and time handling guide](/guide/date-and-time-handling.md).
+   *
+   * @example
+   * E.g. for `timeFormats = ['hh:mm:ss']`, valid time strings include:
```

Comment:
Let's stick to the default formats values to avoid confusing the readers:

```suggestion
   * E.g. for `timeFormats = ['hh:mm:ss.sss']`, valid time strings include:
```

Reply:
Ok. It makes sense.